UnityProject/Assets/Scripts/Items/Weapons/ExplosionUtils.cs

```diff
@@ -20,7 +24,18 @@ public static class ExplosionUtils
 		public static void PlaySoundAndShake(Vector3Int worldPosition, byte shakeIntensity, int shakeDistance)
 		{
 			string sndName = EXPLOSION_SOUNDS[Random.Range(0, EXPLOSION_SOUNDS.Length)];
-			SoundManager.PlayNetworkedAtPos(sndName, worldPosition, -1f, true, true, shakeIntensity, (int)shakeDistance);
+			SoundManager.PlayNetworkedAtPos(sndName, worldPosition, -1f, true, true, shakeIntensity, (int)shakeDistance, false);
```

Comment:
Codacy found an issue: [Remove this unnecessary cast to 'int'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6394885)